src/components/accessories/patientNewBill/styles.scss

```diff
@@ -0,0 +1,118 @@
+@import "../../../styles/variables";
+@import "../../../../node_modules/susy/sass/susy";
+
+.patientNewBill {
+  display: flex;
+  flex-flow: column;
+  border: 1px solid hsl(2, 98%, 53%, .075);
+  border-radius: 8px;
+  box-shadow: 2px 2px 2px hsl(2, 98%, 53%, .075);
+  padding: 16px 8px 16px 8px;
+  row-gap: 1em;
```

Comment:
We already have the "overlay" effect inside the table, we don't need to use another wrapper with the same style.

Reply:
So we have to remove the borther and the elevation effect; right ?

Reply:
Yep